### PR TITLE
NRPT-164: Adding clear keywords button to keyword box

### PR DESCRIPTION
**Pull request by dhlevi**
Adding the clear keyword button to the text box for keyword searches in the record search page.

See ticket [NRPT-164](https://bcmines.atlassian.net/browse/NRPT-164)

### CODE REVIEW

File: angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts [hunk 1]
```diff
@@ -172,6 +172,12 @@ export class RecordsListComponent implements OnInit, OnDestroy {
     });
   }
 
+  /**
+   * Clears the active keyword search text
+   */
+  clearSearchTerms() {
+    this.keywordSearchWords = '';
```
Comment (NickPhura) [inline angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts:179]: Does the `X` button need to trigger the search when clicked?

Comment (dhlevi) [reply angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts:179]: Good question. Not mentioned in the ticket, but kinda makes sense. 

Note that currently that clear advanced search does not re-search either.

Comment (NickPhura) [reply angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts:179]: Oh gotcha.  If the current clear doesn't trigger the search, then seems fine that this wouldn't either. They should at least be consistent with eachother, I think.

Comment (NickPhura) [reply angular/projects/admin-nrpti/src/app/records/records-list/records-list.component.ts:179]: Oh gotcha.  If the current clear doesn't trigger the search, then seems fine that this wouldn't either. They should at least be consistent with eachother, I think.

**Pull request merged**